src/api/app/models/notifications_filter.rb

```diff
@@ -0,0 +1,11 @@
+class NotificationsFilter
```

Comment:
I'm not sure where to put this model

Reply:
This looks like a presenter. A quite basic one, but still a presenter.

Reply:
Hmm, yes I thought about it before, but I wasn't sure. I'll try again.

Reply:
I've renamed the class as a presenter